knowledge-base/handling-errors-occurring-in-a-custom-resolver-used-by-a-reporting-rest-service.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Handling errors occurring in a Custom Resolver used by a Reporting REST Service
+description: Handling errors occurring in a Custom Resolver used by a Reporting REST Service.
```

Comment:
As discussed, change the description - https://wiki.progress.com/display/DTS/Customer+Context